lib/datalog/roletester/proto/access.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package types;
+
+// Facts represents a datalog database of predicates
+message Facts {
```

Comment:
Isn't this already defined in types.proto? Do you need both / any way to avoid duplication?